addons/mass_mailing/models/mailing.py

```diff
@@ -750,6 +759,14 @@ def _prepare_statistics_email_values(self):
             'formatted_date': formatted_date,
         }
 
+    def _get_pretty_mailing_type(self):
+        if self.mailing_type == 'mail':
+            return _('Emails')
+
+    def _get_count_sent(self):
+        if self.mailing_type == 'mail':
```

Comment:
Actually you could directly return self.sent, as parent will take care of not calling super() if mailing type is different. Depends on if we keep this method also (see other comment).